### ISSUE TITLE
I Have Solved The bug of many Programs Not Opening Like Office

### ISSUE THREAD

**Issue by ArielD10** [tjackenpacken/taskbar-groups#184]
You need to go to start and go to the app that you want to put in the group and right-click and click open file location

![Screenshot 2021-09-30 113223](https://user-images.githubusercontent.com/91683593/135487244-9ee73571-a360-439a-ae20-bdcf5c075c5d.png)
 
Then When open right-click in the shortcut and click open file location again

![Screenshot 2021-09-30 113345](https://user-images.githubusercontent.com/91683593/135487687-ecd6a8ca-7ce0-4ea7-b2ad-06f36cfb4787.png)
n 

When open check the location on the file

![Screenshot 2021-09-30 113443](https://user-images.githubusercontent.com/91683593/135487897-c8add0c4-b340-4859-9b08-b5dd64fd2eff.png)
s 

Next, open Taskbargroups and make your group, when it opens file explorer for putting the shortcut,  open the file location that I say to remember

![Screenshot 2021-09-30 113613](https://user-images.githubusercontent.com/91683593/135488354-11484765-4220-4d35-9d4c-1d4f12c79e26.png)
 
And There Will open I hope it has been helpful to you :) This Works with office too 

If there are other problems leave a comment

**Comment by ArielD10** [comment]
You can use it for any program that is download from a file not from the Microsoft store. You can use it with office

![Screenshot 2021-09-30 121405](https://user-images.githubusercontent.com/91683593/135492801-4182851a-3b7f-4204-a747-608dd692a47e.png)
.

**Comment by WahyuAgungBudiyanto** [comment]
@ArielD10 Is this the location for Word? 
C:\Program Files\Microsoft Office\root\Office16

**Comment by WahyuAgungBudiyanto** [comment]
@ArielD10 Thank you it works

**Comment by ArielD10** [comment]
Yes That is the location of word  C:\Program Files\Microsoft Office\root\Office16  
@WahyuAgungBudiyanto

**Comment by ArielD10** [comment]
You welcome!!

**Comment by Diztend** [comment]
How about chrome shortcuts?

**Comment by ArielD10** [comment]
Do The Same @Diztend
